src/WinUI/ArcGISRuntime.WinUI.Viewer/Samples/Layers/DisplayOACollection/readme.metadata.json

```diff
@@ -17,7 +17,7 @@
         "web"
     ],
     "nuget_packages": {
-        "Esri.ArcGISRuntime": "100.12.0"
+        "Esri.ArcGISRuntime": "100.13.0"
```

Comment:
It would be best to exclude these unrelated changes, and maybe open a second PR with metadata cleanup